src/components/Navbar/Navbar.styles.ts

```diff
@@ -1,35 +1,29 @@
 import { createUseStyles } from 'react-jss';
 
+const useNavBarSticky = () => ({
+  background: '#f1f1f1',
+  position: 'fixed',
+  top: 0,
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  padding: '0.5rem 2.5rem',
+  zIndex: '1',
+  width: '-webkit-fill-available',
+});
+
 export const useNavbarStyles = createUseStyles({
   navbar: {
     position: 'absolute',
     top: '-100%',
   },
 
-  navbarSticky: {
```

Comment:
I dry'ed this up a bit, nothing wrong with it. Just noticed a pattern.

Reply:
love this! I forgot we're using JSS and was writing my css in a class specific way. if we were using actual css i could reuse a specific class but React doesn't allow multiple classes on ClassName and we're not using SCSS which i could do a bunch with that. With JSS we can do stuff like this for reuse and i love it. I like having this in it's own file even more 👍🏽